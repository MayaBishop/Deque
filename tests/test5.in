#tests add an item to the front and back of the deque once there is an item in the deque

#start with first item at the front
enqueue_front 1
enqueue_back 2
enqueue_front 3
print
enqueue_back 4
print
clear

#starts with first item at the back
enqueue_back 1
enqueue_front 2
enqueue_back 3
print
enqueue_front 4
print
