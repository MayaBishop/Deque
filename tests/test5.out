success
success
success
3 1 2
2 1 3
success
3 1 2 4
4 2 1 3
success
success
success
success
2 1 3
3 1 2
success
4 2 1 3
3 1 2 4
